expansion/src/main/java/no/unit/nva/expansion/model/ExpandedDoiRequest.java

```diff
@@ -164,11 +165,38 @@ private static ExpandedDoiRequest fromDoiRequest(DoiRequest doiRequest, Resource
         entry.setCustomerId(doiRequest.getCustomerId());
         entry.setModifiedDate(doiRequest.getModifiedDate());
         entry.setOwner(doiRequest.getOwner());
-        entry.setStatus(doiRequest.getStatus());
+        entry.setStatus(getExpandedTicketStatus(doiRequest));
         entry.setViewedBy(doiRequest.getViewedBy());
         entry.setPublication(publicationSummary);
         entry.setFinalizedBy(doiRequest.getFinalizedBy());
         entry.setOwner(doiRequest.getOwner());
         return entry;
     }
+
+    private static ExpandedTicketStatus getExpandedTicketStatus(DoiRequest doiRequest) {
```

Comment:
Extract getTicketStatusParse(), getNewTicketStatus and getExpandedTicketStatus to a common static util class?